profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,125 @@
+variable "plan" {
+  type        = string
+  description = "Plan for the event stream instance. lite, standard or enterprise-3nodes-2tb"
+  default     = "enterprise-3nodes-2tb"
+}
+
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the event stream instance"
+  type        = string
+}
+
+variable "tags" {
+  type        = list(string)
+  description = "List of tags associated with the Event Steams instance"
+  default     = []
+}
+
+variable "es_name" {
+  description = "Name of the event streams instance"
+  type        = string
+}
+
+variable "schemas" {
+  type = list(object(
+    {
+      schema_id = string
+      schema = object({
+        type = string
+        name = string
+      })
+    }
+  ))
+  description = "The list of schema object which contains schema id and format of the schema"
+  default = [{
+    schema_id = "my-es-schema_1"
+    schema = {
+      type = "string"
+      name = "name_1"
+    }
+    },
+    {
+      schema_id = "my-es-schema_2"
+      schema = {
+        type = "string"
+        name = "name_2"
+      }
+    },
+    {
+      schema_id = "my-es-schema_3"
+      schema = {
+        type = "string"
+        name = "name_3"
+      }
+    }
+  ]
+}
+
+variable "topics" {
+  type = list(object(
+    {
+      name       = string
+      partitions = number
+      config     = object({})
+    }
+  ))
+  description = "List of topics. For lite plan only one topic is allowed."
+  default = [
```

Comment:
The fscloud module should not have any topic set by default. So default should be set to `[]`

Reply:
Done